src/main/java/mate/academy/bookstore/service/impl/UserServiceImpl.java

```diff
@@ -15,14 +17,21 @@
 import org.springframework.stereotype.Service;
 
 @Service
-@AllArgsConstructor
+@RequiredArgsConstructor
 public class UserServiceImpl implements UserService {
     private final UserRepository userRepository;
     private final RoleRepository roleRepository;
     private final UserMapper userMapper;
     private final LocaleService localeService;
     private final PasswordEncoder encoder;
 
+    private long userRoleId;
+
+    @PostConstruct
+    void init() {
+        userRoleId = roleRepository.getByName(Role.RoleName.USER).getId();
+    }
+
```

Comment:
OPTIONAL: Not sure that this should be state of service, maybe it makes sense to make some class like UserConfigHolder and put it there to separate service and some config responsibilities